packages/core/core/test/AssetGraphBuilder.test.js

```diff
@@ -0,0 +1,79 @@
+// @flow strict-local
+
+import invariant from 'assert';
```

Comment:
`invariant` has a hardcoded meaning in flow, letting it refine nullable types. In its callable form, it has identical behavior to `assert`, so I'm just aliasing it here.